pkg/template/resource.go

```diff
@@ -50,6 +50,15 @@ func ResolveTrigger(trigger triggersv1.EventListenerTrigger, getTB getTriggerBin
 	tb := make([]*triggersv1.TriggerBinding, 0, len(trigger.Bindings))
 	ctb := make([]*triggersv1.ClusterTriggerBinding, 0, len(trigger.Bindings))
 	for _, b := range trigger.Bindings {
+		if b.Spec != nil {
+			tb = append(tb, &triggersv1.TriggerBinding{
```

Comment:
Don't we need to return out of  this loop here else  we'll try to fetch the binding in getTB and fail?

Reply:
Used continue done.